example_dags/example_transform.py

```diff
@@ -8,17 +8,17 @@
 from astro.sql.table import Table
 
 START_DATE = datetime(2000, 1, 1)
-LAST_ONE_DF = pd.DataFrame(data={"Title": ["Random movie"], "Rating": [121]})
+LAST_ONE_DF = pd.DataFrame(data={"title": ["Random movie"], "rating": [121]})
 
 
 # [START transform_example_1]
 @aql.transform()
-def top_five_animations(input_table: Table):
+def top_five_animations(input_table: Table):  # skipcq: PYL-W0613
```

Comment:
`# skipcq: PYL-W0613` can be removed. I feel @kaxil  has added it globally.

Reply:
@sunank200 mentioned before, this check was failing 3h ago in this PR :(